selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -953,6 +954,31 @@ public boolean isVisible(WebElement webElement) {
     }
   }
 
+  /**
+   * Determines whether or not this element {@link WebElement} is selected (uses for checkboxes,
+   * radio buttons, etc.)
+   *
+   * @param webElement element which should be selected
+   * @return true if given element is selected
+   */
+  public boolean isSelected(WebElement webElement) {
+    try {
+      return webElement.isSelected();
+    } catch (NoSuchElementException ex) {
+      return false;
+    }
+  }
+
+  /**
+   * Determines whether or not this element {@link WebElement} is enabled
+   *
+   * @param webElement element which should be enabled
+   * @return true if given element is enabled
+   */
+  public boolean isEnabled(WebElement webElement) {
```

Comment:
Name doesn't reflect waiting. 
`waitElementIsEnabled()` sounds more relevant, IMHO.

Reply:
The matter is, that the method returns boolean value. It does not wait, it determines the state of the given web-element - enabled or disabled.

Reply:
I see you point, but it actually wait because there is `waitVisibility(webElement).isEnabled()` command in the method body. And it's better to declare it in the name to avoid duplication of code.
How about another name `isVisibleAndEnabled()`

Reply:
OK, for me.